go.mod

```diff
@@ -55,6 +55,8 @@ require (
 	github.com/jrick/logrotate v1.0.0
 	github.com/kylelemons/godebug v1.1.0 // indirect
 	github.com/marcopeereboom/sbox v1.0.0
+	github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd // indirect
```

Comment:
Is it oki that `go.mod` is changing when I'm running `go install` command ?

Reply:
What go version are you using?

Reply:
@lukebp `1.12.7`, should I upgrade to more recent one ?